crates/router/src/core/webhooks/incoming.rs

```diff
@@ -99,6 +99,7 @@ pub async fn incoming_webhooks_wrapper<W: types::OutgoingWebhookType>(
         .attach_printable("Could not convert webhook effect to string")?;
 
     let api_event = ApiEvent::new(
+        state.tenant.tenant_id.clone(),
```

Comment:
If we are only logging this event by passing a reference in `log_event` function. Why do we need to clone?
(Just minor optimization, you can pick this up later if you wish)